ember-cli-build.js

```diff
@@ -3,67 +3,49 @@
 const MergeTrees = require('broccoli-merge-trees');
 const Funnel = require('broccoli-funnel');
 const Rollup = require('broccoli-rollup');
-const Plugin = require('broccoli-plugin');
 const path = require('path');
+const typescript = require('broccoli-typescript-compiler');
+const stew = require('broccoli-stew');
 
-const walkSync = require('walk-sync');
-const fs = require('fs');
-const mkdirp = require('mkdirp').sync;
-
-class TestIndex extends Plugin {
-  constructor(input) {
-    super([input], {
-      persistentOutput: true,
-      annotation: 'tests/index.js'
-    });
-  }
-
-  build() {
-    let inputPath = this.inputPaths[0];
-    let outputPath = this.outputPath + '/tests';
-    let testModules = walkSync(inputPath, {
-      globs: ['**/*.js']
-    }).map(file => {
-      let moduleId = file.slice(0, -3);
-      return `import './${moduleId}';`;
-    }).join('\n');
-    mkdirp(outputPath);
-    fs.writeFileSync(outputPath + '/index.js', testModules);
+const ts = typescript(stew.find('lib'), {
```

Comment:
Updates above are committed. Appears that the 1 ES6 export not being compiled was/is a pre-existing condition on the current `master`. Needs additional exploration.